static/app/views/explore/tables/fieldRenderer.tsx

```diff
@@ -0,0 +1,77 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import Link from 'sentry/components/links/link';
+import TimeSince from 'sentry/components/timeSince';
+import type {EventData, MetaType} from 'sentry/utils/discover/eventView';
+import EventView from 'sentry/utils/discover/eventView';
+import {getFieldRenderer} from 'sentry/utils/discover/fieldRenderers';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {generateLinkToEventInTraceView} from 'sentry/utils/discover/urls';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {TraceViewSources} from 'sentry/views/performance/newTraceDetails/traceMetadataHeader';
+import {getTraceDetailsUrl} from 'sentry/views/performance/traceDetails/utils';
+
+interface FieldProps {
+  data: EventData;
+  dataset: DiscoverDatasets;
+  field: string;
+  meta: MetaType;
+  unit?: string;
+}
+
+export function FieldRenderer({data, dataset, field, meta, unit}: FieldProps) {
```

Comment:
Nice! This is a good start but we should plan on splitting this component up into smaller ones as this component grows bigger.

Reply:
Agree, we can iterate.